src/screens/Home/Home.styles.js

```diff
@@ -1,9 +1,47 @@
 import { StyleSheet } from 'react-native';
 
 export const styles = StyleSheet.create({
-  container: {
+  separator: {
+    height: 1,
+    width: '100%',
+    backgroundColor: '#CCC',
+  },
+  listHeader: {
+    height: 55,
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+
+  listHeadLine: {
+    color: '#333',
+    fontSize: 21,
+    fontWeight: 'bold',
+  },
+
+  item: {
     flex: 1,
+    flexDirection: 'row',
+    alignItems: 'center',
+    paddingVertical: 13,
+  },
+
+  avatarContainer: {
+    borderRadius: 10,
+    height: 89,
+    width: 89,
```

Comment:
Have you taken a look at how your app looks on low-sized device screens?
You should try on multiple devices (with different screen sizes) and see if all of them properly show the layout.